tests/expirationProduct.test.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable */
```

Comment:
This file should be named `product.test` and must contain all product-related tests.